setenv.sh

```diff
@@ -27,3 +27,6 @@ export TORQPROCESSES=${KDBAPPCONFIG}/process.csv
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$KDBLIB/l32
 
 export TORQMONIT=${TORQHOME}/logs/monit                                                             # set the folder for monit outputs
+
+export RLWRAP=QCON=/usr/bin/rlwrap                                                                  # set rlwrap path
```

Comment:
Won't this set `RLWRAP` to `QCON=/usr/bin/rlwrap`, is that right?

Reply:
This is a typo, Fixed it!